api/db/migrate/20210907133742_user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :uid, null: false, unique: true
+      t.string :name, null: false
+      t.string :bio, null: false
+      t.string :profile_image,default: ''
```

Comment:
ここはnull okやからdefaultはいいかな
```suggestion
      t.string :profile_image
```